_includes/signup.html

```diff
@@ -1,7 +1,5 @@
-<div class="sign-up-container">
-  <a href="https://runnable.com/signup/" class="well well-anchor sign-up-box grid-block col track-sign-up noscroll">
-    <img class="img no-touching" src="{{ site.url }}/images/runnabear-hello.png">
-    <p class="p">Try Runnable! It’s <strong class="weight-strong">free</strong> for 14 days.
-    <div data-target="#sign-up" class="btn btn-sm green js-modal track-sign-up">Sign Up</div>
-  </a>
-</div>
+<a href="https://runnable.com/signup/" class="sign-up-box grid-block col well well-anchor track-sign-up">
+  <img class="img no-touching" src="{{ site.url }}/images/runnabear-hello.png">
+  <p class="p">Try Runnable! It’s <strong class="weight-strong">free</strong> for 14 days.
```

Comment:
For some reason this isn't showing up on my screen. Should I insert gray-dark for the well-anchor? I'll keep looking out for the gray text.